.github/actions/maven-publish/action.yml

```diff
@@ -36,9 +36,9 @@ runs:
     - name: Publish Java
       shell: bash
       if: inputs.is-android == 'false'
-      run: ./gradlew clean assemble sign publishMavenJavaPublicationToMavenRepository -PisSnapshot=false -Pversion="${{ inputs.version }}" -PossrhUsername="${{ inputs.ossr-username }}" -PossrhPassword="${{ inputs.ossr-password }}" -PsigningKey="${{ inputs.signing-key }}" -PsigningPassword="${{ inputs.signing-password }}"
+      run: ./gradlew clean assemble sign publishMavenJavaPublicationToMavenRepository -PisSnapshot=false -Pversion="${{ inputs.version }}" -PossrhUsername="${{ inputs.ossr-username }}" -PossrhPassword="${{ inputs.ossr-token }}" -PsigningKey="${{ inputs.signing-key }}" -PsigningPassword="${{ inputs.signing-password }}"
 
     - name: Publish Android
       shell: bash
       if: inputs.is-android == 'true'
-      run: ./gradlew clean assemble sign publishAndroidLibraryPublicationToMavenRepository -PisSnapshot=false -Pversion="${{ inputs.version }}" -PossrhUsername="${{ inputs.ossr-username }}" -PossrhPassword="${{ inputs.ossr-password }}" -PsigningKey="${{ inputs.signing-key }}" -PsigningPassword="${{ inputs.signing-password }}"
```

Comment:
This commands wouldn't work as we are passing `ossr-token` as value to the `ossrhPassword` parameter. You may refer [here](https://github.com/auth0/Auth0.Android/blob/cea3c17f2f506d9d320b1c2c59bcf5343ee60bf3/.github/actions/maven-publish/action.yml#L35) for what we are doing in Auth0.android

Furthermore, we've simplified these two steps into a single step which can work for both Android and Java packages,

Reply:
Updated the commands, in reference with auth0-android